### DISCUSSION TITLE
where should I report bug

### DISCUSSION THREAD

**Discussion by Liujiahao0** [qpdf/qpdf#996] [Q&A]
Excuse me, I found some code problems during the Fuzz of the qpdf. If I want to determine whether this is an error, where should I report it,and where can I view the errors that have been discovered so that I can determine if the problem I want to report has been found

**Comment by jberkenbilt** [comment]
If you believe it is a security-sensitive bug, you can report it to me by private email at ejb@ql.org. Otherwise, feel free to just post an issue. If you're not sure, you're welcome to just email me. It may take me some time to respond, but I will respond. If you include `qpdf` in the subject, it will help ensure that the message sorts into my qpdf mailbox.
